templates/admin/index.html.twig

```diff
@@ -0,0 +1,96 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}JobPermut Admin
```

Comment:
didn't you do a fix changing "JobPermut" to "Jobpermut"? (with lowercase "p")

Reply:
Yes i've do this fix, but I forgot this. Sorry